share/wake/lib/core/print.wake

```diff
@@ -100,3 +100,18 @@ export def printlnLevel (LogLevel name) (message: String): Unit =
 #   def Unit = println "{integerToUnicode 0x1f600}"
 export def println: String => Unit =
     printlnLevel logReport
+
+# breadcrumb: Leaves an out of band message in the wake internal log
+#
+# This should primarily be used by core/standard libraries over normal user code.
+# However it can be useful for tracing or debugging wake code out of band. The contents
+# of the log may only be inspected outside of wake and thus any breakcrumbs are
+# "blackholed" from the perspective of wakelang.
```

Comment:
Do/can we prevent the log file from being `claim`-ed?

Reply:
Not at the moment. I'm not decided if we actually needed to go as far as enforcing it in code  or if we can enforce via PR.

If nothing else, we could likely just claim it in the stdlib blocking it from being claimed by anyone else

Reply:
I think this is in the same camp as claim-ing wake.db...bad shit is gonna happen if you do it but we don't stop anyone from doing it as far as I'm aware.